.gitignore

```diff
@@ -64,7 +64,6 @@ coverage.xml
 # Force .gitignore files
 !.gitignore
 .noseids
-Procfile
```

Comment:
I think you can safely keep that line. Future procfile from developpers using it won't be commited.